### ISSUE TITLE
AC-GAN gives better performance than the AG-GAN

### ISSUE THREAD

**Issue by TanmDL** [Heye-SYSU/AGGAN#1]
Hi, thank you for sharing the code. I have tested your algorithm performance with ACGAN algorithm. The training dataset has been taken as [2000:class-5, 200:class-6] and the testing dataset is [500:class-5, 500:class-6]. The obtained results for both the algorithms are given below,
AC-GAN: best accuracy-  MAX: 0.8940 (1k epochs)
AG-GAN: best accuracy: Max: 86.7 (1k epochs)

I have tried two cases. In both cases, AC-GAN gives better performance than the AG-GAN.

**Comment by TanmDL** [comment]
Did you take the different architecture for discriminator of AC-GAN algorithm? I have tried with the same architecture for both algorithms. Please look into this matter.
